lib/UI/common.dart

```diff
@@ -0,0 +1,141 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:chrome_management_app/models/keys_util.dart';
+import 'package:flutter/material.dart';
+
+/// Widget to show a custom text line.
+Widget textLine(
```

Comment:
You could make this an actual class, a bit nicer than doing an inline function.

You can do this.

class TextLine extends Padding {
  TextLine({Key key, String test, TextStyle style, EdgeInsetsGeometry geometry) : super(
 padding: padding,
  child: Text(text, maxLines: 2, key: key, overflow: TextOverflow.ellipsis, style: style); }

Just as easy and a class is way easier to deal with.  Same for the rest of these methods.

Reply:
I would do this as methods if you are inside a class, but top level global a class makes more sense.

Reply:
Sure, I just did the change, thanks for the advise.